src/index.js

```diff
@@ -504,46 +504,46 @@
                         .on('paused', err => {
                             if (err) {
                                 vm.$emit('pouchdb-changes-error', {
-                                    db: localDB,
+                                    db: db,
                                     error: err,
                                 });
                                 return;
                             }
                             numPaused += 1;
                             if (numPaused >= 2) {
                                 vm.$emit('pouchdb-changes-paused', {
-                                    db: localDB,
+                                    db: db,
```

Comment:
Not sure where db is supposed to come from since you only have localDB and remoteDB